src/main/java/org/tron/common/utils/ByteUtil.java

```diff
@@ -721,4 +723,40 @@ public static int numberOfLeadingZeros(byte[] bytes) {
     return parseBytes(input, offset + 32 * idx, 32);
   }
 
+  public static boolean greater(byte[] bytes1, byte[] bytes2) {
+    return compare(bytes1, bytes2) > 0;
+  }
+
+  public static boolean greaterOrEqual(byte[] bytes1, byte[] bytes2) {
+    return compare(bytes1, bytes2) >= 0;
+  }
+
+  public static boolean less(byte[] bytes1, byte[] bytes2) {
+    return compare(bytes1, bytes2) < 0;
+  }
+
+  public static boolean lessOrEqual(byte[] bytes1, byte[] bytes2) {
+    return compare(bytes1, bytes2) <= 0;
+  }
+
+  public static boolean equal(byte[] bytes1, byte[] bytes2) {

```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either override Object.equals(Object obj), or totally rename the method to prevent any confusion. ([squid:S1221](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1221))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1159&issues=AWSs-S_MIUupByjQEIf-&open=AWSs-S_MIUupByjQEIf-)

[ISSUE_KEY]: AWSs-S_MIUupByjQEIf-